src/app/user/register/register.component.ts

```diff
@@ -7,9 +9,22 @@ import { Component, OnInit } from '@angular/core';
 })
 export class RegisterComponent implements OnInit {
 
-  constructor() { }
+  constructor(private userService: UserService) { }
+  registerInfo = {
+    username: "",
+    email: "",
+    password: "",
+    confirm_password: "",
+  }
 
   ngOnInit(): void {
   }
 
+  register(): void {
```

Comment:
set the userinfo (check userService) and navigate user to home page so the user don't have to login again once he/she registered